media/client/main/source/WebAudioPlayer.cpp

```diff
@@ -22,6 +22,7 @@
 #include "IWebAudioPlayerIpcClient.h"
 #include "RialtoClientLogging.h"
 #include <cstring>
+#include <iostream>
```

Comment:
I suppose you don't need this include ;)

Reply:
Removed! (and I found some other places where iostream was included... also removed them)